### PR TITLE
Yaw issue102 curent user details

### LINKED ISSUE #102: Get Current Login User
- Endpoint URI: */users/current*
- Http Method: *GET*
- API Logic
  - Get current logic user based on Security
  - Map with User Club Family
    - Name
    - Slogan? or Description?

- Progress
    - [x] development
    - [x] integration Test

### PR DESCRIPTION
**Pull request by HowardYaw**
resolved #102

### CODE REVIEW

File: src/main/resources/config/liquibase/data/club_family_TestFaker.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+id;name;slogan
+1;Five Fishes;Five_Fishes_Give_Me_Five
```
Comment (kachun333) [inline src/main/resources/config/liquibase/data/club_family_TestFaker.csv:2]: Give me ✋ 🤣

**Review by kachun333 — DISMISSED** [review]

File: src/main/java/com/thirdcc/webapp/web/rest/UserResource.java [hunk 2]
```diff
@@ -177,6 +184,26 @@ public UserResource(UserService userService, UserRepository userRepository, Mail
...
+            .orElseThrow(() -> new BadRequestException("User not Login"));
+        UserDTO userDTO = userService.getUserByLogin(userLogin)
+            .map(UserDTO::new)
+            .orElseThrow(() -> new BadRequestException("User not found"));
+        userCCInfoService.findOneByUserId(userDTO.getId())
+            .ifPresent(userCCInfoDTO -> userService.mapUserWithUserCCInfo(userDTO, userCCInfoDTO));
+        log.debug("User DTO after map: {}", userDTO);
+        return ResponseEntity.ok().body(userDTO);
+    }
+
```
Comment (LUXIANZE) [inline src/main/java/com/thirdcc/webapp/web/rest/UserResource.java]: @HowardYaw can we do this in GET: /users/:login ? What is the intention of creating this endpoint?

Comment (HowardYaw) [reply src/main/java/com/thirdcc/webapp/web/rest/UserResource.java]: we can do in that also
but using this endpoint API consumer no need to pass in any `login`
we use the SecurityUtils to get current/logged in user login

File: src/main/resources/config/liquibase/data/club_family_TestFaker.csv (hunk 1 above)
Comment (HowardYaw) [reply src/main/resources/config/liquibase/data/club_family_TestFaker.csv:2]: ✋

**Review by CarlosSia — APPROVED** [review]

**Review by junzai97 — APPROVED** [review]

**Pull request merged**